app/about/the-team/page.tsx

```diff
@@ -1,7 +1,86 @@
 "use client";
+import { useGroupedTeamMemberData } from "@/hooks/useGroupedTeamMemberData";
```

Comment:
> Overall think the changes made a huge difference but there are a bunch of layout concerns we will need to address soon! and yes I am ok with opening a seperate PR for that (and ticket ofc)

Yeah that sounds good to me man, I think some kind of "design review" ticket would be good. We could get some feedback in basecamp on some of the copy on the different pages as well :)